src/Middleware/HttpLogging/test/HttpLoggingOptionsTests.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace Microsoft.AspNetCore.HttpLogging.Tests
```

Comment:
idk about runtime, but we usually put the tests in the same namespace as the type being tested.